### DISCUSSION TITLE
Profile Image on card layout

### DISCUSSION THREAD

**Discussion by janetslei** [microsoft-search/pnp-modern-search#2733] [Q&A]
Is there anything I need to change in the default settings of the search results web part to get the profile picture of the Author to show up in the card layout? The People layout shows it with no issues for the same user, what am I missing?
![image](https://user-images.githubusercontent.com/8562215/216224914-d57d704a-effa-4e50-8749-62e3c239c795.png)

**Comment by wobba** [comment]
This is a bug, and created an issue for it https://github.com/microsoft-search/pnp-modern-search/issues/2743
